task/orchestrator.go

```diff
@@ -277,13 +283,24 @@ func (ctx *OrchestrationContext) internalScheduleActivity(activityName string, o
 		},
 	}
 
+	// Add TaskRouter support for cross-app activities
+	if options.targetAppID != nil && ctx.appID != nil {
+		router := &protos.TaskRouter{
+			Source: *ctx.appID,           // Current orchestrator app ID
+			Target: *options.targetAppID, // Target activity app ID
```

Comment:
It does seem like these should be pointers (?).

Reply:
Based on the protos I have:
```
message TaskRouter {
    string source = 1; // orchestrationAppID
    string target = 2; // appID
}
```

Which means that is a string. Are you wanting me to change that to something else? I changed the options and ctx appID to be *string per your request? The task router is optional, not the source & target inside the task router.